ilp/api_urls.py

```diff
@@ -0,0 +1,26 @@
+"""ilp URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/1.11/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  url(r'^$', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  url(r'^$', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.conf.urls import url, include
+    2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
+"""
+from django.conf.urls import url, include
+from boundary.views import Admin1sBoundary
+state_patterns = [
+    url(r'^boundary/admin1s$', Admin1sBoundary.as_view(), name='admin1s-boundary'),
+    url(r'^boundary/admin2s$',Admin1sBoundary.as_view(), name='admin2s-boundary'),
+    url(r'^boundary/admin3s$',Admin1sBoundary.as_view(), name='admin3s-boundary')
+]
+urlpatterns = [
+    url(r'^(?P<state>[a-zA-z]{2})/', include(state_patterns))
+]
+
```

Comment:
Also, try to keep all urls corresponding to each app in the respective app itself instead of having one big api_urls.py file.